model_analyzer/result/result_manager.py

```diff
@@ -319,9 +365,10 @@ def _add_results_to_heaps(self):
             'ResultManager.results')
 
         for model_name in self._analysis_model_names:
-            model_measurements = results.get_model_measurements_dict(model_name)
-
-            if not model_measurements:
+            try:
```

Comment:
IS this correct? Does it now raise instead of returning None if it does not exist?

Reply:
can you add a unit test to confirm if there is not one already/

Reply:
We (inadvertently) have multiple unit tests covering this case. I've fixed the unit tests so that they run profile instead of analysis, before adding in their measurements.